Source/Client/Sound/DirectSound.cs

```diff
@@ -87,7 +94,7 @@ public static bool Initialize(Form target)
 			BuildLog10Table();
 
 			// Get settings from configuration
-			playeffects = General.config.ReadSetting("sounds", true);
```

Comment:
Wait a minute, don't we need that actually? This is a setting that should turn off the sounds.

Reply:
This setting is enabled by default in the launcher